impeller/renderer/backend/gles/texture_gles.cc

```diff
@@ -21,19 +22,20 @@ static TextureGLES::Type GetTextureTypeFromDescriptor(
   const auto usage = static_cast<TextureUsageMask>(desc.usage);
   const auto render_target =
       static_cast<TextureUsageMask>(TextureUsage::kRenderTarget);
-  if (usage == render_target) {
+  const auto is_msaa = desc.sample_count == SampleCount::kCount4;
+  if (usage == render_target && !is_msaa) {
     return TextureGLES::Type::kRenderBuffer;
```

Comment:
Maybe file a bug to investigate if we should investigate RenderBuffer multisampled (now that we have something working)

Reply:
Added.